### PR TITLE
Feature/amb price history graph

### PR DESCRIPTION
**Pull request by JavidHaji-zada**

### CODE REVIEW

File: src/components/templates/BezierChart/data.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  // Take a random sample using inverse transform sampling method.
+  return distribution.ppf(Math.random());
+}
+
+export function generateRandomGraphData(length: number): GraphPoint[] {
+  return Array<number>(length)
+    .fill(0)
+    .map((_, index) => ({
+      date: new Date(
+        new Date(2000, 0, 1).getTime() + 1000 * 60 * 60 * 24 * index
```
Comment (ilyaAir) [inline src/components/templates/BezierChart/data.ts:14]: what do we get here?

File: src/components/templates/BezierChart/index.tsx [hunk 2]
```diff
@@ -1,62 +1,104 @@
...
+        SelectionDot={SelectionDot}
       />
-    </VictoryChart>
+      {intervals.length > 0 && (
+        <>
+          <Row
+            alignItems="center"
+            justifyContent="space-between"
+            style={{
+              width: '80%',
```
Comment (ilyaAir) [inline src/components/templates/BezierChart/index.tsx:94]: maybe Stylesheet

**Review by ilyaAir — APPROVED** [review]
in general LGTM!

File: src/components/templates/BezierChart/data.ts (hunk 1 above)
Comment (JavidHaji-zada) [reply src/components/templates/BezierChart/data.ts:14]: it is dummy, will remove it.

**Pull request merged**